lib/features/mailbox/data/model/mailbox_not_destroyed_type.dart

```diff
@@ -0,0 +1,3 @@
+enum MailBoxNotDestroyedType {
+  notFound,
+}
```

Comment:
`jmap-dart-client` have not defined the type for this error before?

Reply:
@hoangdat No